presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/BasePlanTest.java

```diff
@@ -207,6 +211,29 @@ protected void assertPlanValidatorWithSession(@Language("SQL") String sql, Sessi
         });
     }
 
+    protected void assertPlanFailedWithExceptionWithSession(String sql, Session session, @Language("RegExp") String regex)
```

Comment:
this method name is pretty long.  I don't think you need "WithSession" in the method name.  That's conveyed when you pass in a session.  Also, instead of calling the field regex, call it `expectedException` or `expectedExceptionRegex`

Reply:
Done